src/website/app/pages/ComponentStatus/componentStatus.md

```diff
@@ -17,29 +17,66 @@ Statuses:
-| [Button](/components/button)                | [available]      |
-| [Card](/components/card)                    | [available]      |
-| [Checkbox](/components/checkbox)            | [available]      |
-| [Dropdown](/components/dropdown)            | [available]      |
-| [Flex](/components/flex)                    | [available]      |
-| [Form](/components/form-field)              | [available]      |
-| [Icon](/components/icon)                    | [available]      |
-| [Link](/components/link)                    | [available]      |
-| [Menu](/components/menu)                    | [available]      |
-| [Popover](/components/popover)              | [available]      |
-| [Radio](/components/radio)                  | [available]      |
-| [Select](/components/select) ~ new          | [available]      |
-| [StartEnd](/components/start-end)           | [available]      |
-| [TextArea](/components/text-area)           | [available]      |
-| [TextInput](/components/text-input)         | [available]      |
-| [ThemeProvider](/components/theme-provider) | [available]      |
-| [Tooltip](/components/tooltip)              | [available]      |
-| Grid                                        | [planned]        |
-| Toggle                                      | [planned]        |
-| DataTable                                   |                  |
-| DatePicker                                  |                  |
-| Navigation                                  |                  |
+| Component                                   | Status      |
+|---------------------------------------------|-------------|
+| [Avatar](/components/avatar)                | [available] |
+| [Box](/components/box)                      | [available] |
+| [Button](/components/button)                | [available] |
+| [Card](/components/card)                    | [available] |
+| [Checkbox](/components/checkbox)            | [available] |
+| [Dropdown](/components/dropdown)            | [available] |
+| [Flex](/components/flex)                    | [available] |
+| [Form](/components/form-field)              | [available] |
+| [Icon](/components/icon)                    | [available] |
+| [Link](/components/link)                    | [available] |
+| [Menu](/components/menu)                    | [available] |
+| [Popover](/components/popover)              | [available] |
+| [Radio](/components/radio)                  | [available] |
+| [Select](/components/select) ~ new          | [available] |
+| [StartEnd](/components/start-end)           | [available] |
+| [TextArea](/components/text-area)           | [available] |
+| [TextInput](/components/text-input)         | [available] |
+| [ThemeProvider](/components/theme-provider) | [available] |
+| [Tooltip](/components/tooltip)              | [available] |
+| Grid                                        | [planned]   |
+| Toggle                                      | [planned]   |
```

Comment:
Should we move Dialog up here & add the `planned` status since it's been groomed & in the next sprint?

Reply:
Same for Text?

Reply:
I think we'll address these next week since we haven't committed to anything yet